app/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     compile "org.springframework.boot:spring-boot-starter-web:${springBootVersion}"
     compile "org.springframework.boot:spring-boot-starter-thymeleaf:${springBootVersion}"
     compile "org.springframework.boot:spring-boot-starter-data-jpa:${springBootVersion}"
+    compile "org.springframework.boot:spring-boot-devtools:${springBootVersion}"
```

Comment:
Não to ligado no que isso faz, mas parece útil.

Reply:
https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-devtools.html